.github/workflows/CI.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6'
+          - '1.10'  # lts
```

Comment:
Couldn't we just use lts here directly? We might have to change the julia-action version but that should be possible